components/schedule-editor/src/ScheduleEditor.svelte

```diff
@@ -47,37 +48,66 @@
       event_description,
       "",
     );
+    show_hosts = getPropertyValue(internalProps.show_hosts, show_hosts, "show");
   }
 
-  $: editableProperties = [
-    {
-      label: "Event Title",
-      value: event_title,
-    },
-    {
-      label: "Event Description",
-      value: event_description,
-    },
-  ];
+  $: manifestProperties = {
+    event_title,
+    event_description,
+    show_hosts,
+  };
   // #endregion mount and prop initialization
 
   function saveProperties() {
     console.log("Saving the following properties:");
-    console.log(editableProperties);
+    Object.entries(manifestProperties).forEach(([k, v]) => {
+      console.log(k, v);
+    });
   }
 </script>
 
 <style lang="scss">
+  fieldset {
+    margin: 0;
+    padding: 0;
+    border: 0;
+  }
 </style>
 
 {#if manifest && manifest.error}
   <nylas-domain-error {id} />
 {:else}
-  {#each editableProperties as prop}
+  <fieldset>
+    <label>
+      <strong>Event Title</strong>
+      <input type="text" bind:value={manifestProperties.event_title} />
+    </label>
+  </fieldset>
+  <fieldset>
+    <label>
+      <strong>Event Description</strong>
+      <input type="text" bind:value={manifestProperties.event_description} />
+    </label>
+  </fieldset>
+  <fieldset>
+    <strong>Show meeting hosts to the end-user?</strong>
+    <label>
+      <input
+        type="radio"
+        bind:group={manifestProperties.show_hosts}
+        value="show"
+      />
+      <span>Show Hosts</span>
```

Comment:
Do the `<span>` tags add any styling here?

Reply:
Not currently, but I expect we'll want some text-wrapping element for when we do.